src/main/scala/com/github/unchama/seichiassist/data/HalloweenItemData.scala

```diff
@@ -1,25 +1,32 @@
 package com.github.unchama.seichiassist.data
 
-import scala.collection.immutable.{List, Set}
-import scala.jdk.CollectionConverters._
-import scala.util.chaining._
-import java.util._
+import java.util.Calendar
 
+import com.github.unchama.seichiassist.util.Util
 import de.tr7zw.itemnbtapi.NBTItem
-import org.bukkit.Bukkit
-import org.bukkit.ChatColor
+import org.bukkit.ChatColor._
 import org.bukkit.Color.fromRGB
+import org.bukkit.{Bukkit, Material}
 import org.bukkit.enchantments.Enchantment
-import org.bukkit.Material
+import org.bukkit.inventory.meta.PotionMeta
 import org.bukkit.inventory.{ItemFlag, ItemStack}
-import org.bukkit.inventory.meta.{ItemMeta, PotionMeta}
 import org.bukkit.potion.{PotionEffect, PotionEffectType}
 
+import scala.collection.immutable.{List, Set}
+import scala.jdk.CollectionConverters._
+import scala.util.chaining._
+
 object HalloweenItemData {
-  def getHalloweenPotion(): ItemStack = {
-    val potionMeta: PotionMeta = Bukkit.getItemFactory.getItemMeta(Material.POTION).asInstanceOf[PotionMeta]
-      .tap(_.setDisplayName(s"${ChatColor.AQUA}${ChatColor.ITALIC}うんちゃまの汗"))
+
+  /*
+  HalloweenPotionここから
+   */
+
+  def getHalloweenPotion: ItemStack = {
```

Comment:
```suggestion
  val halloweenPotion: ItemStack = {
```
`val` で良さそう？ (`EventCommand` の方も要rename)